### ISSUE TITLE
Ktor 2 support

### ISSUE THREAD

**Issue by sschuberth** [ermadmi78/kobby#24]
Hi, are there any plans to support Ktor 2 (which is now at version 2.1.0)?

**Comment by ermadmi78** [comment]
Hi. I agree, it's time to release Kobby 2.0 version, with Ktor 2 support. Additionally, in version 2.0 it is necessary to raise the minimum requirements for the Kotlin version from 1.5 to 1.6. 
I'll try to do it in the fall. If you need Ktor 2 support as soon as possible, I would recommend turning off the default adapter generation and implementing the adapter interface manually:
```kotlin
plugins {
    kotlin("jvm") version "1.6.21"
    id("io.github.ermadmi78.kobby") version "1.5.1"
}

kobby {
    kotlin {
        adapter {
            ktor {
                simpleEnabled = false
                compositeEnabled = false
            }
        }
    }
}
```

**Comment by ermadmi78** [comment]
See an example of a manually implemented Ktor 2 adapter [here](https://github.com/ermadmi78/kobby-gradle-example/tree/ktor2/cinema-api/src/main/kotlin/io/github/ermadmi78/kobby/cinema/api/kobby/kotlin/adapter/ktor)

**Comment by ermadmi78** [comment]
Available since release 2.0.0-beta.01
